.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 python:
-    - "2.6"
```

Comment:
If supports for 2.6 is really dropped, this worths a separated PR with doc updates

Reply:
and what about this one?

Reply:
It's the travis support of python 2.6 which is dropped, not the milkcheck one.
Without this commit, travis complains.

Reply:
aaaah ok